utils/frame/remote-externalities/src/lib.rs

```diff
@@ -139,20 +156,33 @@ impl<B: BlockT> Default for OnlineConfig<B> {
 	fn default() -> Self {
 		Self {
 			transport: Transport { uri: DEFAULT_TARGET.to_owned(), client: None },
+			scrape_children: false,
 			at: None,
 			state_snapshot: None,
 			pallets: vec![],
 		}
 	}
 }
 
+impl<B: BlockT> From<String> for OnlineConfig<B> {
+	fn from(s: String) -> Self {
+		Self { transport: s.into(), ..Default::default() }
```

Comment:
I would actually expect to scrape children by default.
Like scraping the entire chain except child tries is a bit surprising

Reply:
fair enough, will revert.